diff --git a/dev-requirements.txt b/dev-requirements.txt
index ef12307..1d65750 100644
--- a/dev-requirements.txt
+++ b/dev-requirements.txt
@@ -1 +1,3 @@
 pyyaml>=4.2b4
+-e fusion-common
+-e cloudify-utilities-plugins-sdk
diff --git a/setup.py b/setup.py
index 841a362..cb724bd 100644
--- a/setup.py
+++ b/setup.py
@@ -17,7 +17,7 @@ import os
 import re
 import pathlib
 
-from setuptools import setup
+from setuptools import setup, find_packages
 
 
 def get_version():
@@ -35,7 +35,7 @@ setup(
     author='Cloudify Platform LTD',
     author_email='hello@cloudify.co',
     description='Manage Ansible nodes by Cloudify.',
-    packages=['cloudify_ansible', 'cloudify_ansible_sdk'],
+    packages=find_packages(),
     package_data={
         'cloudify_ansible': [
             'ansible-cloudify-ctx/modules/cloudify_runtime_property.py',
@@ -45,8 +45,8 @@ setup(
     license='LICENSE',
     zip_safe=False,
     install_requires=[
-        "cloudify-common>=4.5.5",
-        "cloudify-utilities-plugins-sdk>=0.0.121",
+        "fusion-common",
+        "cloudify-utilities-plugins-sdk>=0.0.129",
         "ansible",
         "pexpect==4.8.0",
         "networkx",